scripts/datacontainer/run.sh

```diff
@@ -0,0 +1,15 @@
+
+#!/bin/bash
+
+timestamp=`date +%Y/%m/%d-%H:%M:%S`
+echo "[$timestamp] Copying files to /usr/data/openapispecs"
+mkdir -p /usr/data/openapispecs/.git
+cp -R /usr/data/specrepo/* /usr/data/openapispecs/
```

Comment:
cp -R /usr/data/specrepo /usr/data/openapispecs should be sufficient. The cp in linux is different with COPY in Dockerfile.

Reply:
Oh sorry you've already makdir for openapispecs...